src/inc/clrnt.h

```diff
@@ -916,7 +916,7 @@ RtlpGetFunctionEndAddress (
     if ((FunctionLength & 3) != 0) {
         FunctionLength = (FunctionLength >> 2) & 0x7ff;
     } else {
-        FunctionLength = *(PTR_ULONG)(ImageBase + FunctionLength) & 0x3ffff;
+        FunctionLength = *(PTR_ULONG)((SIZE_T)(ImageBase + FunctionLength)) & 0x3ffff;
```

Comment:
Would it be better to change the ImageBase argument to `TADDR` ?

Reply:
Yes it's better indeed, thanks